core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/toDataFrame.kt

```diff
@@ -60,6 +60,21 @@ class CreateDataFrameTests {
         res["e"].type() shouldBe typeOf<Int>()
     }
 
+    @Test
+    fun `create column with infer type`() {
+        val data: List<Any> = listOf(1, 2, 3)
+        val res = data.toDataFrame {
+            "e" from inferType { it }
```

Comment:
same here, it's good it exists, but I think it should be done by default when using `expr`